### PR TITLE
v2v: support unpriviliged user run v2v command

### PR DESCRIPTION
**Pull request by xiaodwan**
Add support for running v2v command in unpriviliged user.

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.env_leak.esx_67.json.default.it_vddk.vpx_uri: PASS (709.64 s)

**Comment by xiaodwan** [comment]
depends on https://github.com/autotest/tp-libvirt/pull/3387

**Comment by xiaodwan** [comment]
@chloerh @yantian0 Please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -6,15 +6,16 @@
 
 from __future__ import print_function
 
-import os
-import re
-import time
+import aexpect
```
Comment (chloerh) [inline virttest/utils_v2v.py:9]: Why is it so above?

Comment (xiaodwan) [reply virttest/utils_v2v.py:9]: Just make the module in alphanumeric order.

Comment (chloerh) [reply virttest/utils_v2v.py:9]: Oh, ok. But `Imports should be grouped in the following order: Standard library imports. Related third party imports.` 
So maybe we  could put it a little bit lower. But nice arrangement. I could tell that you have also read the slides of Clean Code.

**Review by yantian0 — APPROVED** [review]

File: virttest/utils_v2v.py (hunk 1 above)
Comment (xiaodwan) [reply virttest/utils_v2v.py:9]: oh, I forgot that. Let me update it. 
Thanks.

**Review by chloerh — APPROVED** [review]

**Pull request merged**